README.md

```diff
@@ -2,51 +2,93 @@
 [![CI](https://github.com/argoproj/argo-workflows/workflows/CI/badge.svg)](https://github.com/argoproj/argo-workflows/actions?query=event%3Apush+branch%3Amaster)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3830/badge)](https://bestpractices.coreinfrastructure.org/projects/3830)
 
-![Argo Image](docs/assets/argo.png)
```

Comment:
I feel like we should keep the logo?

Reply:
it pushes a lot of content below the fold - and we have the logo on the page elsewhere